python/features/calc/model/MathFunction.yaml

```diff
@@ -7,5 +7,3 @@ enum:
     - DIVIDE
     - POWER
     - SQUARE_ROOT
-  validation:
```

Comment:
So, because of the new `primitive_constraint` implementation, the enum check is handled differently and doesn't need to be explicitly applied anymore, that's why this `validation` portion was able to be removed?

Reply:
In principle, yet...but this is an Enum and not a primitive, so it's handled a little differently.

Reply:
So, mis-categorization on my part of the enum type when I first commented. The additional check of the structure and entries of the enum is now being enforced as part of the `load_definitions()` in the `language_context` within the `create_enum_class()`, if I am following things better now?